### PR TITLE
camelize drop output column names and update gnu-wget container

### PR DESCRIPTION
**Pull request by Lucpen**
## PR checklist

- [X] camelize drop output column names and update gnu-wget container
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/genomic-medicine-sweden/tomte/tree/master/.github/CONTRIBUTING.md)
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: bin/drop_filter_results.py [hunk 1]
```diff
@@ -148,14 +151,27 @@ def filter_fraser_result(
...
     )
 
 
+def camelize_dataframe_columns(df):
+    def camelize(string):
+        if match("^[a-z]+([A-Z][a-z0-9]*)*$", string):
+            return string
+        parts = split("([^a-zA-Z0-9])", string)
+        camelized = "".join(part.capitalize() for part in parts if part.isalnum())
+        return camelized[0].lower() + camelized[1:]
```
Comment (jemten) [inline bin/drop_filter_results.py:169]: We could add a fallback to return the original string in the case where the regex doesn't match. 
Chatgpt provided me with this:
```python
 # Return camelized string or original if empty
return camelized[0].lower() + camelized[1:] if camelized else string
```

**Review by jemten — COMMENTED** [review]
A few quick comments

File: bin/drop_filter_results.py [hunk 2]
```diff
@@ -114,8 +116,9 @@ def filter_outrider_results(
     df_family_annotated_aberrant_expression_top_hits = annotate_with_drop_gene_name(
         df_family_aberrant_expression_top_hits, out_drop_gene_name
     )
+    # df_family_annotated_aberrant_expression_top_hits = camelize_dataframe_columns(df_family_annotated_aberrant_expression_top_hits)
```
[the remark this answers is not in the document]
Comment (Lucpen) [reply bin/drop_filter_results.py]: Yes :)

File: bin/drop_filter_results.py [hunk 3]
```diff
@@ -148,14 +151,27 @@ def filter_fraser_result(
...
+    camelize_dataframe_columns(df_results_family_aberrant_splicing).to_csv(
         file_name_research, sep="\t", index=False, header=True
     )
     filter_by_gene_panel(
         df_results_family_aberrant_splicing, gene_panel, file_name_research
     )
 
 
+def camelize_dataframe_columns(df):
+    def camelize(string):
```
[the remark this answers is not in the document]
Comment (Lucpen) [reply bin/drop_filter_results.py]: Done

File: bin/drop_filter_results.py [hunk 4]
```diff
@@ -148,14 +151,27 @@ def filter_fraser_result(
...
+    # df_results_family_aberrant_splicing = camelize_dataframe_columns(df_results_family_aberrant_splicing)
+    camelize_dataframe_columns(df_results_family_aberrant_splicing).to_csv(
         file_name_research, sep="\t", index=False, header=True
     )
     filter_by_gene_panel(
         df_results_family_aberrant_splicing, gene_panel, file_name_research
     )
 
 
+def camelize_dataframe_columns(df):
```
[the remark this answers is not in the document]
Comment (Lucpen) [reply bin/drop_filter_results.py]: Done

File: bin/drop_filter_results.py (hunk 1 above)
Comment (Lucpen) [reply bin/drop_filter_results.py]: Done

**Review by jemten — APPROVED** [review]
🐫 🎉

**Pull request merged**